Formula/m/manim.rb

```diff
@@ -22,21 +22,34 @@ class Manim < Formula
   depends_on "cython" => :build
   depends_on "ninja" => :build
   depends_on "pkg-config" => :build
+
   depends_on "cairo" # for cairo.h
   depends_on "ffmpeg"
   depends_on "fontconfig"
   depends_on "freetype"
   depends_on "glib"
+  depends_on "jpeg-turbo"
+  depends_on "libtiff"
+  depends_on "little-cms2"
   depends_on "numpy"
+  depends_on "openblas"
   depends_on "pango"
   depends_on "pillow"
   depends_on "py3cairo"
   depends_on "python@3.12"
   depends_on "scipy"
 
+  uses_from_macos "zlib"
+
   on_macos do
+    depends_on "gcc"
```

Comment:
Might be able to get rid of this if you remove the `scipy` resource and use the formula instead.